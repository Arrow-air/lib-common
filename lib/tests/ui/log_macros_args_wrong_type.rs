//! Test too many arguments for log_macros

use lib_common::log_macros;

log_macros!(not_a_StrLit);

fn main() {}
