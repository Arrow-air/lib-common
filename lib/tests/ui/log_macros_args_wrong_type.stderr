error: expected string literal
 --> tests/ui/log_macros_args_wrong_type.rs:5:13
  |
5 | log_macros!(not_a_StrLit);
  |             ^^^^^^^^^^^^
